src/event.js

```diff
@@ -649,16 +649,12 @@ jQuery.event = {
 	},
 
 	simulate: function( type, elem, event, bubble ) {
-		// Piggyback on a donor event to simulate a different one.
-		// Fake originalEvent to avoid donor's stopPropagation, but if the
-		// simulated event prevents default then we do the same on the donor.
```

Comment:
I think most of this comment still applies and should be kept. Perhaps you could change "Fake originalEvent" to "Don't set originalEvent" instead?